.github/workflows/build-and-test.yaml

```diff
@@ -1,6 +1,9 @@
 name: build-and-test-scheduled
 
 on:
+  push:
+    branches:
+      - tier4/main
```

Comment:
There is no branch named `tier4/main`.
We only have `tier4/universe`(temporary) and `main` here.

Reply:
Oh, thanks.

Reply:
Fixed branch name. 9deab047